test/cmd_line/read.test.ts

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+import { ModeHandler } from '../../src/mode/modeHandler';
+import { setupWorkspace, cleanUpWorkspace, assertEqualLines } from './../testUtils';
+import { runCmdLine } from '../../src/cmd_line/main';
+
+suite("read", () => {
+  let modeHandler: ModeHandler;
+
+  setup(async () => {
+    await setupWorkspace();
+    modeHandler = new ModeHandler();
+  });
+
+  teardown(cleanUpWorkspace);
+
+  test("Can read shell command output", async () => {
+    await runCmdLine('r! echo "hey ho"', modeHandler);
```

Comment:
I just didn't realise that Windows prints the "s in echo "..". I modified the test so that it passes on Windows too.